examples/rust/standalone/README.md

```diff
@@ -0,0 +1,25 @@
+# Simple Round-Trip Example
```

Comment:
This looks like a duplicate of a README one folder down and should probably be removed.